### PR TITLE
Updating company-cadence.md

### PR DESCRIPTION
**Pull request by natalie-hub**
Updated Staff Enablement survey cadence

### CODE REVIEW

**Comment by natalie-hub** [comment]
Hi @justinegeffen - I've discussed these changes with Ian. The changes are based on COM survey feedback from last week.

File: operations/operations/company-cadence.md [hunk 1]
```diff
@@ -111,10 +111,11 @@ After the prep meeting, Vice Chair posts a screenshot of the final agenda into t
 
 Every 6 months we'll be asking staff to spend around 3 minutes completing a staff enablement survey of 12 engagement questions, plus identifying their organizational leader. ‌
 
-* Survey is announced in COM meeting
+* Enablement Survey will be announced in COM meeting 1-week before the survey goes live
```
Comment (justinegeffen) [inline operations/operations/company-cadence.md]:
```suggestion
* Enablement Survey will be announced in COM meeting one week before the survey goes live.
```

File: operations/operations/company-cadence.md [hunk 2]
```diff
@@ -111,10 +111,11 @@ After the prep meeting, Vice Chair posts a screenshot of the final agenda into t
...
+* Survey link will be shared at the beginning of COM
```
Comment (justinegeffen) [inline operations/operations/company-cadence.md]:
```suggestion
* The survey link will be shared at the beginning of COM.
```

File: operations/operations/company-cadence.md [hunk 3]
```diff
@@ -111,10 +111,11 @@ After the prep meeting, Vice Chair posts a screenshot of the final agenda into t
 
 Every 6 months we'll be asking staff to spend around 3 minutes completing a staff enablement survey of 12 engagement questions, plus identifying their organizational leader. ‌
 
-* Survey is announced in COM meeting
+* Enablement Survey will be announced in COM meeting 1-week before the survey goes live
+* Survey link will be shared at the beginning of COM 
 * Link is posted in COM channel with "Thumbs Up" emoji reaction
```
Comment (justinegeffen) [inline operations/operations/company-cadence.md]:
```suggestion
* The link is posted in COM channel with "Thumbs Up" emoji reaction.
```

File: operations/operations/company-cadence.md [hunk 4]
```diff
@@ -111,10 +111,11 @@ After the prep meeting, Vice Chair posts a screenshot of the final agenda into t
...
 * All staff are asked to complete the survey and signal completion by clicking on Thumbs Up
```
Comment (justinegeffen) [inline operations/operations/company-cadence.md]:
```suggestion
* All staff are asked to complete the survey and signal completion by clicking on "Thumbs Up".
```

File: operations/operations/company-cadence.md [hunk 5]
```diff
@@ -111,10 +111,11 @@ After the prep meeting, Vice Chair posts a screenshot of the final agenda into t
...
-* Meeting is concluded after Thumbs Up count reaches attendee count
+* Ideally, the meeting will conclude after Thumbs Up count reaches attendee count but this is not a requirement.
```
Comment (justinegeffen) [inline operations/operations/company-cadence.md]:
```suggestion
* Ideally, the meeting will conclude after "Thumbs Up" count reaches attendee count but this is not a requirement. 
```

File: operations/operations/company-cadence.md [hunk 6]
```diff
@@ -111,10 +111,11 @@ After the prep meeting, Vice Chair posts a screenshot of the final agenda into t
...
 Every 6 months we'll be asking staff to spend around 3 minutes completing a staff enablement survey of 12 engagement questions, plus identifying their organizational leader. ‌
 
-* Survey is announced in COM meeting
+* Enablement Survey will be announced in COM meeting 1-week before the survey goes live
+* Survey link will be shared at the beginning of COM 
 * Link is posted in COM channel with "Thumbs Up" emoji reaction
 * All staff are asked to complete the survey and signal completion by clicking on Thumbs Up
-* Meeting is concluded after Thumbs Up count reaches attendee count
+* Ideally, the meeting will conclude after Thumbs Up count reaches attendee count but this is not a requirement. 
 * An analysis of the results should be prepared and presented in the next 2-4 weeks in COM
```
Comment (justinegeffen) [inline operations/operations/company-cadence.md]:
```suggestion
* An analysis of the results should be prepared and presented within the next 2 - 4 weeks in COM.
```

File: operations/operations/company-cadence.md [hunk 7]
```diff
@@ -111,11 +111,12 @@ After the prep meeting, Vice Chair posts a screenshot of the final agenda into t
 
 Every 6 months we'll be asking staff to spend around 3 minutes completing a staff enablement survey of 12 engagement questions, plus identifying their organizational leader. ‌
 
-* Survey is announced in COM meeting
-* Link is posted in COM channel with "Thumbs Up" emoji reaction
-* All staff are asked to complete the survey and signal completion by clicking on Thumbs Up
-* Meeting is concluded after Thumbs Up count reaches attendee count
-* An analysis of the results should be prepared and presented in the next 2-4 weeks in COM
+* Enablement Survey will be announced in COM meeting one week before the survey goes live.
```
Comment (justinegeffen) [inline operations/operations/company-cadence.md]:
```suggestion
* The enablement survey will be announced in COM one week before the survey goes live.
```

File: operations/operations/company-cadence.md [hunk 8]
```diff
@@ -111,11 +111,12 @@ After the prep meeting, Vice Chair posts a screenshot of the final agenda into t
...
+* The survey link will be shared at the beginning of COM.
+* The link is posted in COM channel with "Thumbs Up" emoji reaction.
```
Comment (justinegeffen) [inline operations/operations/company-cadence.md]:
```suggestion
* The link will be posted in the COM channel with "Thumbs Up" emoji reaction.
```

**Review by justinegeffen — APPROVED** [review]
Thanks, this looks good. :)

**Pull request merged**